core/imds/src/main/java/software/amazon/awssdk/imds/Ec2MetadataClientBuilder.java

```diff
@@ -33,15 +34,30 @@
     /**
      * Define the retry policy which includes the number of retry attempts for any failed request.
      * <p>
-     * If not specified, defaults to 3 retry attempts and a {@link BackoffStrategy#defaultStrategy()}  backoff strategy} that
-     * uses {@link RetryMode#STANDARD}
+     * If not specified, defaults to 3 retry attempts and a {@link BackoffStrategy#defaultStrategy()} backoff strategy} that
+     * uses {@link RetryMode#STANDARD}. Can be also specified by using the
+     * {@link Ec2MetadataClientBuilder#retryPolicy(Consumer)} method. If both methods are called multiple time, the last call
```

Comment:
A bit ambiguous here. Do you mean something like, "If either method is called multiple times, or if both methods are called" ?
Same for below

Reply:
Yeah that sounds good to me